Geometry/GEMGeometryBuilder/src/GEMGeometryParsFromDD.cc

```diff
@@ -20,6 +20,7 @@
 
 #include <iostream>
 #include <algorithm>
+#include <boost/lexical_cast.hpp>
```

Comment:
@jshlee - please, remove dependency on boost, prefer to use the standard library instead.